k8s/workloads/pastebin/pastebin-ingress.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ingress-nginx
+  namespace: pastebin-prod
+  labels:
+    app: pastebin
+spec:
+  releaseName: pastebin-ingress-nginx
+  chart:
+    repository: https://kubernetes.github.io/ingress-nginx
+    name: ingress-nginx
+    version: "3.33.0"
```

Comment:
I could be looking at the wrong thing but I don't see a tag for `ingress-nginx` with this tag but I do see `heml-chart` with this tag. Does the `name` here map to `helm-chart` instead of `ingress-nginx`?

Reply:
@bkochendorfer 3.33.0 is the version / release of the ingress-nginx helm chart: https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-3.33.0

Reply:
ah, so it somehow maps the name correctly here with some helm magic?